native/src/widget/scrollable.rs

```diff
@@ -18,6 +18,8 @@ pub struct Scrollable<'a, Message, Renderer: self::Renderer> {
     scroller_width: u16,
     content: Column<'a, Message, Renderer>,
     style: Renderer::Style,
+    on_scroll: Option<Box<dyn Fn(f32, f32) -> Message>>,
```

Comment:
It might be better to have `Fn(f32, f32) -> Message` to be a generic param rather than a box/dynamic dispatch?

I guess this is up to @hecrj, but ive only seen `Box<dyn xxx>` used in `Element`, and generics used everywhere else.

Reply:
Yeah I'm not sure about that, just did the first thing that came into my mind. Will wait for @hecrj's review then.